mkt/search/tests/test_views.py

```diff
@@ -1346,6 +1346,18 @@ def test_search(self):
         eq_(objs[1]['title'], self.website.title)
         eq_(objs[1]['url'], self.website.url)
 
+    def test_search_preferred_region_match(self):
+        """
+        For websites, if the query string is something that will definitely not
+        match any websites we may still match on preferred_region. But we don't
+        want only preferred_region to find results.
+        """
+        self.website
```

Comment:
hehe. oops. I was going to add a preferred region and then just used what was there. Good catch.